core/src/main/java/hudson/model/AbstractProject.java

```diff
@@ -751,8 +751,13 @@ protected void updateTransientActions() {
         for (JobProperty<? super P> p : Util.fixNull(properties))
             ta.addAll(p.getJobActions((P)this));
 
-        for (TransientProjectActionFactory tpaf : TransientProjectActionFactory.all())
-            ta.addAll(Util.fixNull(tpaf.createFor(this))); // be defensive against null
+        for (TransientProjectActionFactory tpaf : TransientProjectActionFactory.all()) {
+            try {
+                ta.addAll(Util.fixNull(tpaf.createFor(this))); // be defensive against null
+            } catch (Exception e) {
```

Comment:
Generally style guides will say to catch `RuntimeException` to make it clear that there is no checked exception you are expecting, but I do not think it really matters.